packages/core/src/tools/should-paginate.js

```diff
@@ -0,0 +1,43 @@
+const { get } = require('lodash');
+
+// this is (annoyingly) mirrored in cli/api_base, so that test functions only
+// have a storeKey when canPaginate is true. otherwise, a test would work but a
+// poll on site would fail. this is only used in test handlers
+
+// there are 4 places you can put a method that can interact with cursors:
+// triggers.contact.operation.perform, if it's a poll trigger
+// triggers.contact.operation.performList, if it's a hook trigger
+// resources.contact.list.operation.perform if it's a resource
+// resources.contact.hook.operation.performList if it's a resource
+
+const shouldPaginate = (appRaw, method) => {
+  const methodParts = method.split('.');
+  const methodName = methodParts.pop();
+  const operation = get(appRaw, methodParts);
+
+  if (methodParts[0] === 'triggers') {
+    if (operation.type === 'poll' && methodName === 'perform') {
```

Comment:
🚨  the operation type is `polling`, not `poll` ([docs](https://github.com/zapier/zapier-platform/blob/master/packages/schema/docs/build/schema.md#properties-11))

Reply:
also, the `type` is optional on polling triggers, but (mostly) required on hook triggers (unless they're part of a resource). I don't believe this block (or the tests) cover the missing `type` for non-resource polls.